var/spack/repos/builtin/packages/archer/package.py

```diff
@@ -30,12 +30,12 @@ class Archer(Package):
     """ARCHER, a data race detection tool for large OpenMP applications."""
 
     homepage = "https://github.com/PRUNERS/ARCHER"
+    url='https://github.com/PRUNERS/archer/archive/v1.0.0.tar.gz'
```

Comment:
@alalazo @tgamblin How did this line pass the flake8 tests? Shouldn't it be required to have spaces around the '='?